server/app/views/admin/AdminLayout.java

```diff
@@ -101,12 +122,35 @@ private ContainerTag renderNavBar() {
     String intermediaryLink = routes.TrustedIntermediaryManagementController.index().url();
     String apiKeysLink = controllers.admin.routes.AdminApiKeysController.index().url();
 
+    String activeNavStyle =
+        StyleUtils.joinStyles(
+            BaseStyles.TEXT_SEATTLE_BLUE,
+            Styles.FONT_MEDIUM,
+            Styles.BORDER_B_2,
+            BaseStyles.BORDER_SEATTLE_BLUE);
+
+    NavPage activePage = activeNavPage.orElseThrow();
```

Comment:
The decision to throw felt preferable to having an "unspecified" enum and not highlighting to me in that the page would fail to render when a layout value hadn't been specified. I assessed callsites to make sure we were setting it (and hopefully tests will exercise all of these paths).

Reply:
Suggestion (blocking): an alternative that would ensure it's set without potentially blocking the admin's ability to use the system could be to introduce:
```java
public class AdminLayoutFactory {
  @Inject
  public AdminLayoutFactory(ViewUtils viewUtils, Config configuration) {...}
  
  public AdminLayout getLayout(NavPage navPage) {
    return new AdminLayout(viewUtils, configuration, navPage);
  }
}
```

and then change:
```java
public class AdminLayout extends BaseHtmlLayout {
  
  // make the constructor package-private, remove the @Inject annotation, and require the navPage arg
  AdminLayout(ViewUtils viewUtils, Config configuration, NavPage navPage)
}
```

thoughts?

Reply:
also instead of hand-writing the factory could instead use https://github.com/google/guice/wiki/AssistedInject

Reply:
Done. I shied away from using AssistedInject since we don't make very heavy use of Guice and the amount of boilerplate for the factory was relatively low.